.github/workflows/e2e-validation.yml

```diff
@@ -4,14 +4,19 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - main
+      - feat/add-more-synthetics
```

Comment:
I don't think we want this right @johnrwatson ?

Reply:
Also, ANY push to main will trigger these tests - that doesn't feel right as that code won't have made production yet - should this be limited to any changes to the cypress tests?